### DISCUSSION TITLE
Regional Prompting error

### DISCUSSION THREAD

**Discussion by WhoseBoatIsThisBoat** [vladmandic/sdnext#3498] [General]
so, I activate the regional prompting script, set mode "Prompt", set the prompt threshold to 0.5,0.5 and enter my two prompts separated by BREAK

the results don't look like I'd wish, and I get this error:

```
01:53:46-836325 WARNING  Pipeline class change failed: type=DiffusersTaskType.IMAGE_2_IMAGE
                         pipeline=RegionalPromptingStableDiffusionPipeline AutoPipeline can't find a pipeline linked to
                         RegionalPromptingStableDiffusionPipeline for None
```

any thoughts on what to do next?

thanks!

**Comment by vladmandic** [comment]
need a full log to say for sure, but seems. like you tried to use it in img2img and regional prompting is only supported with txt2img.

**Reply by WhoseBoatIsThisBoat** [reply]
How do I get you a full log?

I'm in the Text tab, I haven't done any img2img as yet.

**Reply by vladmandic** [reply]
you upload it here.
btw, you can be in text tab, but it really depends what you were doing - for example, hires is img2img operation.

**Reply by WhoseBoatIsThisBoat** [reply]
I get that I upload it here, but what is it that you want for a log? The command-line output while running --debug, or?

**Reply by vladmandic** [reply]
yes

**Reply by WhoseBoatIsThisBoat** [reply]
For some reason, as I was trying to reproduce the issue, it's no longer there and I was able to trial-and-error my way into getting it to work.

Still, thank you for helping out - much appreciated!
